addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -380,6 +380,10 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
         // Ribbons may have been edited or deleted in another products' option, need to make sure they're up to date
         this.rerender = true;
     },
+    onBlur: function () {
+        this._resetRibbonDummy();
```

Comment:
Not sure why this reset on blur?

Reply:
This is to prevent from getting a very incoherent result when saving a page if you edited a bunch of ribbons without validating, since as long as you have not validated the ribbon, it will do nothing on save. The other possibility would be to implicitly save the ribbon onBlur but that seemed more destructive to me.

Reply:
Oh ok. Probably better to document that or to use `cleanForSave` method for it which is explicitly made for that :+1:

Reply:
The idea was that we don't let the user edit the same ribbon in multiple places without visual feedback that their changes will have no effect on save. There is nothing to do in cleanForSave as this is just changing the option's internal state but nothing will be saved until validated regardless. This is just a way to communicate that to the user visually.